### ISSUE TITLE
Consider versioning models directly in the repo

### ISSUE THREAD

**Issue by erinys** [onnx/models#632]
# Feature Request

### What is the problem that this feature solves?
Current Git LFS path and GDrive model download paths are ok, but painful to actually figure out what you actually want, having to download everything locally or go file by file and then loading into Netron.

### Describe the feature
Replace Git LFS and GDrive by just adding models and datasets directly into the repo with the [XetData app and extension](https://github.com/apps/xetdata). This allows for easier updates and integrated Netron views. I made a [copy of the existing repo with models](https://github.com/xetdata/onnx-models) so you can see what this would look like. It's a nicer development experience as well, since we show before and after views on commits and PRs.
![netron-demo](https://github.com/onnx/models/assets/1296134/c8cb227c-9ef3-449c-b589-ebd0987f37d9)

### Relevant Model Zoo feature areas
**Feature Area**: Model storage, file organization

### Notes
I work for XetData and we are free for all public usage. Happy to make a PR for these updates if there's interest. We also have a version hosted with built in views on [XetHub](https://xethub.com/XetHub/onnx-models).

**Comment by ramkrishna2910** [comment]
Hi @erinys 
Thanks for opening the issue. 
The integrated Netron visualization is really interesting.
If the models are not stored on Git LFS as its done now, where do the onnx files actually live?

**Comment by erinys** [comment]
The models would live in the XetHub storage backend, which is backed by S3 and free for all public repositories.

**Comment by srinify** [comment]
Here's an example @ramkrishna2910

- Pointer file in GitHub: https://github.com/xetdata/onnx-models/blob/main/vision/style_transfer/fast_neural_style/model/candy-9.onnx
- Link to XetHub hosted view of the file: https://xethub.com/integration/github/view?owner=xetdata&repo=onnx-models&ref=main&path=vision/style_transfer/fast_neural_style/model/candy-9.onnx

**Comment by ramkrishna2910** [comment]
Hi @srinify @erinys 
Can we setup a meeting to discuss the above proposal?

**Comment by erinys** [comment]
Definitely! Let me know when you're free or grab some time on my calendar: https://meetings.hubspot.com/annhuang/book.

**Comment by JigishaM** [comment]
Commenting to get updates
